### PR TITLE
Changes for new LVDT, aperture 4, X-motion.

### PR DESCRIPTION
**Pull request by mkestra**
This PR is to save changes that were made for the X-motion of Aperture 4 on AT1K0. the only changes here are to set the soft limits of the system for the newly calibrated LVDT and change the expected nominal point of the X-motion for PMPS. Latest commit was to run pre-commit and some significant formatting changes resulted in multiple files.

## Motivation and Context
The old LVDT on Aperture 4 was sticking so we replaced it with a new one and recalibrated.

## How Has This Been Tested?
Verified with alignment that these parameters are sufficient when attempting to reach hard limits, soft limits, null point, and beam-center.

## Where Has This Been Documented?
On a Jira ticket, https://jira.slac.stanford.edu/browse/LCLSECSD-1897

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive comments
- [x] Test suite passes locally
- [x] Libraries are set to fixed versions and not ``Always Newest``
- [x] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

**Comment by mkestra** [comment]
@ghalym this has now been pre-commit checked and should be ready for review. Please note a lot of, strictly formatting file changes from pre-commit.

File: plc-kfe-gatt/plc_kfe_gatt/PlcTask.TcTTO [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
   <Task Name="PlcTask" Id="{89597bf6-9d0c-45d7-a65b-c0b844a84c6a}">
     <!--CycleTime in micro seconds.-->
     <CycleTime>10000</CycleTime>
     <Priority>20</Priority>
     <PouCall>
       <Name>PRG_1_PlcTask</Name>
     </PouCall>
+    <PouCall>
+      <Name>PRG_VGP</Name>
+    </PouCall>
```
Comment (ghalym) [inline plc-kfe-gatt/plc_kfe_gatt/PlcTask.TcTTO:12]: I am curious why the PRG_VGP is now called with the task?

Comment (mkestra) [reply plc-kfe-gatt/plc_kfe_gatt/PlcTask.TcTTO:12]: That is unclear to me. I didn't edit directly, so I'm not sure what automatic function of build/deploy process would have done this. Is it an issue or is this more just curiosity?

**Comment by mkestra** [comment]
@ghalym I have just committed with a fresh run of pre-commit check and it now passes all checks. Kindly review and approve if appropriate? I need to start working on changes for the roughing pump configuration.

File: plc-kfe-gatt/plc_kfe_gatt/PlcTask.TcTTO (hunk 1 above)
Comment (ghalym) [reply plc-kfe-gatt/plc_kfe_gatt/PlcTask.TcTTO]: looks fixed now

**Review by ghalym — APPROVED** [review]

**Pull request merged**